packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -87,14 +91,23 @@ const labelBaseStyles = ({ theme }: StyleProps) => css`
     content: '';
     display: block;
     position: absolute;
-    top: 50%;
+    top: calc(${theme.typography.body.one.lineHeight} / 2);
     left: ${theme.spacings.bit};
     transform: translateY(-50%) scale(0, 0);
     opacity: 0;
     transition: transform ${theme.transitions.default},
       opacity ${theme.transitions.default};
   }
 `;
+const helperTextBaseStyles = ({ theme }: StyleProps) => css`
+  color: var(--cui-fg-subtle);
+  display: inline-block;
```

Comment:
Changing this to `display: block` would enable you to remove the `<br>` element between the label and description.